app/Models/Concerns/HasFilters.php

```diff
@@ -475,17 +484,34 @@ protected function filterRaces(Builder $query, null|string|array $value = null):
         if (!is_array($value)) {
             $value = [$value];
         }
+        $raceIds = [];
+        foreach ($value as $v) {
+            $raceIds[] = (int) $v;
+        }
 
         if ($this->filterOption('exclude')) {
-            $raceIds = [];
-            foreach ($value as $v) {
-                $raceIds[] = (int) $v;
-            }
             $query->whereRaw('(select count(*) from character_race as cr where cr.character_id = ' .
                 $this->getTable() . '.id and cr.race_id in (' . implode(', ', $raceIds) . ')) = 0');
             return;
         }
 
+        if ($this->filterOption('children')) {
+            $races = DB::select("
```

Comment:
Hardcoding sql is a perfect way to get this to crash on self-hosted Kanka or if we migrate to postgres or another sql engines. We have this all available inside of eloquent.